unidock_tools/README.md

```diff
@@ -11,20 +11,19 @@ Now, UniDockTools has two new functions:
 
 # Installation
 
-## 1. install Uni-Dock and UniDockTools
-UniDock and UniDockTools have been uploaded to the Anaconda repository. To install them, please use the following command:
+## 1. install UniDockTools
```

Comment:
I think installing Uni-Dock first is required. You can add a link to docs of Uni-Dock.